flink-end-to-end-tests/test-scripts/test_streaming_gcp_pubsub.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+cd "${END_TO_END_DIR}/flink-connector-gcp-pubsub-emulator-tests"
+
+mvn test -DskipTests=false
```

Comment:
By default the tests are not skipped, right?

Reply:
They are, in the pom this is explicitly added to prevent a normal `mvn package` from running these long running integration tests.

Reply:
I just noticed the tests are not running any more. `mvn surefire:test` runs them but `mvn test` does not. Do you have any idea why this might be?

Reply:
for me, the tests didn't execute:
```
➜  flink-connector-gcp-pubsub-emulator-tests git:(pr6594) ✗ mvn test -DskipTests=false
[INFO] Scanning for projects...
[INFO]
[INFO] -----< org.apache.flink:flink-connector-gcp-pubsub-emulator-tests >-----
[INFO] Building flink-connector-gcp-pubsub-emulator-tests 1.9-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-checkstyle-plugin:2.17:check (validate) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce-maven-version) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce-maven) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce-versions) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO]
[INFO] --- directory-maven-plugin:0.1:highest-basedir (directories) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO] Highest basedir set to: /Users/robert/Projects/flink
[INFO]
[INFO] --- maven-remote-resources-plugin:1.5:process (process-resource-bundles) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO]
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/robert/Projects/flink/flink-end-to-end-tests/flink-connector-gcp-pubsub-emulator-tests/src/main/resources
[INFO] Copying 3 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.0:compile (default-compile) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO] No sources to compile
[INFO]
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] Copying 3 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.0:testCompile (default-testCompile) @ flink-connector-gcp-pubsub-emulator-tests ---
[INFO] Nothing to compile - all classes are up to date
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  7.389 s
[INFO] Finished at: 2019-05-02T16:47:04+02:00
[INFO] ------------------------------------------------------------------------
```

Reply:
After a clean recompile the tests does not run for me either.